packages/salesforcedx-vscode-core/src/commands/templates/forceApexClassCreate.ts

```diff
@@ -22,8 +25,29 @@ import {
 } from '../util';
 import { OverwriteComponentPrompt } from '../util/postconditionCheckers';
 import { BaseTemplateCommand } from './baseTemplateCommand';
+import { LibraryBaseTemplateCommand } from './libraryBaseTemplateCommand';
 import { APEX_CLASS_DIRECTORY, APEX_CLASS_TYPE } from './metadataTypeConstants';
 
+export class LibraryForceApexClassCreateExecutor extends LibraryBaseTemplateCommand<
+  DirFileNameSelection
+> {
+  public executionName = nls.localize('force_apex_class_create_text');
+  public telemetryName = 'force_apex_class_create';
```

Comment:
@xyc last change, we need to use a different telemetry name for the library or we won't be able to compare events between library executions or normal cli ones.

Reply:
That was my initial thought, but then I wasn't feeling like using a different name for every single template, so I added a custom dimension `commandExecutor` and set it to `library` for all library commands. https://github.com/forcedotcom/salesforcedx-vscode/pull/2428/files#diff-81cf0fdec768a9dede6cce86cb33ffc6R85

Reply:
cool, that should do it